controller.py

```diff
@@ -135,20 +136,27 @@ def run(self):
 
         # Ensure the tier exists.
         metadata = {"order": NET_POL_TIER_ORDER}
-        self._client.set_policy_tier_metadata(NET_POL_TIER_NAME, metadata)
+        self._client.set_policy_tier_metadata("default", metadata)
 
-        # Ensure the backstop policy exists.  This policy fowards
+        # Ensure the backstop policy exists.  This policy forwards
         # any traffic to Kubernetes pods which doesn't match another policy
         # to the next-tier (i.e the per-namespace Profiles).
         selector = "has(%s)" % K8S_NAMESPACE_LABEL
         rules = Rules(inbound_rules=[Rule(action="next-tier")],
                       outbound_rules=[Rule(action="next-tier")])
-        self._client.create_policy(NET_POL_TIER_NAME,
+        self._client.create_policy("default",
                                    "k8s-policy-no-match",
                                    selector,
                                    order=NET_POL_BACKSTOP_ORDER,
                                    rules=rules)
 
+        # Remove old tier if it exists
```

Comment:
Let's actually move this up above the bit where we create the new tier.

I'm not sure what the behavior is for two tiers with the same order - it might make Felix freak out :/